webview/src/experiments/components/table/Table.test.tsx

```diff
@@ -318,25 +318,36 @@ describe('Table', () => {
     })
 
     it('should resize columns and persist new state when a separator is clicked and dragged', async () => {
+      const columnWidths = {
+        id: 333
+      }
+
       const tableDataWithColumnSetting: TableData = {
         ...sortingTableDataFixture,
-        columnWidths: {
-          id: 333
-        }
+        columnWidths
       }
       render(<ExperimentsTable tableData={tableDataWithColumnSetting} />)
       const [experimentColumnResizeHandle] = await screen.findAllByRole(
         'separator'
       )
 
       fireEvent.mouseDown(experimentColumnResizeHandle, {
-        bubbles: true,
-        clientX: 0
+        bubbles: true
+      })
+      fireEvent.mouseUp(experimentColumnResizeHandle)
+
+      expect(mockedPostMessage).toBeCalledWith({
+        payload: { id: 'id', width: 333 },
+        type: MessageFromWebviewType.RESIZE_COLUMN
       })
-      fireEvent.mouseMove(document, {
```

Comment:
[F] Seems like this no longer works because the DOM has no size.

Reply:
Test is changed to accommodate not being a able to move the mouse (still tests the functionality just less directly)